packages/desktop-client/src/components/LoggedInUser.js

```diff
@@ -12,7 +14,7 @@ import { useServerURL } from './ServerContext';
 import { Tooltip } from './tooltips';
 
 export default function LoggedInUser({ hideIfNoServer, style, color }) {
-  let userData = useSelector(state => state.userData);
```

Comment:
The selector is changing this from `state.userData` to `state.user.userData` because there is no `state.userData`.